docs/user/intro.rst

```diff
@@ -3,20 +3,62 @@
+Falcon apps work with any WSGI server, and run great under
+CPython 2.7, PyPy, and CPython 3.3+.
 
-Third, Falcon eschews magic. When you use the framework, it's pretty obvious which inputs lead to which outputs. Also, it's blatantly obvious where variables originate. All this makes it easier to reason about the code and to debug edge cases in large-scale deployments of your application.
+How is Falcon different?
+------------------------
 
-.. _`six`: http://pythonhosted.org/six/
-.. _`mimeparse`: https://code.google.com/p/mimeparse/
+We designed Falcon to support the demanding needs of large-scale cloud
+services and responsive app backends. Falcon complements more general
+Python web frameworks by providing bare-metal performance, reliability,
+and flexibility wherever you need it.
+
+**Fast.** Same hardware, more requests. Falcon turns around
+requests several times faster than most other Python frameworks. For
+an extra speed boost, Falcon compiles itself with Cython when
+available, and also works well with `PyPy <https://pypy.org>`__.
+Considering a move to another programming language? Benchmark with
+Falcon + PyPy first.
+
+**Reliable.** We go to great lengths to avoid introducing
+breaking changes, and when we do they are fully documented and only
+introduced (in the spirit of
+`SemVer <http://semver.org/>`__) with a major version
+increment. The code is rigorously tested with numerous inputs and we
+require 100% coverage at all times. Six and mimeparse are the only
+third-party dependencies.
+
+**Flexible.** Falcon leaves a lot of decisions and implementation
+details to you, the API developer. This gives you a lot of freedom to
+customize and tune your implementation. Due to Falcon's minimalist
+design, Python community members are free to independently innovate on
+Falcon add-ons and complimentary packages. You can also extend the
+framework yourself in a variety of ways.
+
+**Debuggable.** Falcon eschews magic. When you use the framework, it's
+obvious which inputs lead to which outputs, and where variables
+originate. Any potentially surprising behaviors, such as automatic
+request body parsing, are well-documented and disabled by default.
+Falcon doesn't mask exceptions. All this makes it easier to reason about
+the code and to debug edge cases in large-scale deployments. Finally,
+when it comes to the framework itself, we've taken care to keep logic
+paths simple and understandable so that if there ever is a problem, the
+cause can be quickly found and fixed.
```

Comment:
You have a section for "debuggable" here, but not in the readme. Is this intentional?